JotunnLib/Configs/RoomConfig.cs

```diff
@@ -0,0 +1,92 @@
+using Jotunn.Entities;
+
+namespace Jotunn.Configs
+{
+    /// <summary>
+    ///     Configuration class for adding custom clutter.<br />
+    ///     Use this in a constructor of <see cref="CustomRoom"/>
+    /// </summary>
+    public class RoomConfig
+    {
+        /// <summary>
+        ///     Theme name of this room.
+        /// </summary>
+        public string ThemeName { get; }
+
+        /// <summary>
+        ///     <see cref="Room.Theme"/> of this room.
+        /// </summary>
+        public Room.Theme Theme { get; }
```

Comment:
Instead of having both ThemeName and Theme, it would be more suitable to only use the string externally and convert internally to `Room.Theme` when needed

Reply:
I kept both in order to allow devs to add additional rooms to existing dungeons.  Since Room.Theme is always needed (either with one of the vanilla values, or 0 for None), it made sense to me to make themeName optional instead.

I could also do this and hide the Theme behind a string parse, and then when applying the themeName, attempt to parse the string as the enum, but at the time it seemed more complicated and unnecessary.

Reply:
Fixed in 788010f7c6958a35aca8ec43c8579491b8a63ab4